src/olympia/addons/templates/addons/impala/details.html

```diff
@@ -322,6 +322,43 @@ <h4 class="author">{{ _('by') }} {{ users_list(addon.listed_authors) }}</h4>
       </div>
     </div>
   {% endif %}
+  {% if waffle.switch('webext-permissions') %}
+    <div class="modal" id="webext-permissions">
+      <a href="#" class="close">{{ _('close') }}</a>
+      <h2>{{ _('Permissions') }}</h2>
+      {% if addon.current_version and addon.current_version.all_files[0].is_webextension %}
+        {% set permissions = addon.current_version.all_files[0].webext_permissions %}
+        <div class="prose">
+          <p>
+          {{ _('Some add-ons ask for permissions to perform certain functions (example: a tab management add-on will ask permission to access your browser’s tab system).') }}
+          </p><p>
+          {{ _('Since you’re in control of your Firefox, the choice to grant or deny these requests is yours. Accepting permissions does not inherently compromise your browser’s performance or security, but in some rare cases risk may be involved.') }}
+          </p>
+        </div>
+        <div>
+          <p>
+            <h3>{{ _('It can:') }}</h3>
```

Comment:
Since the string is very short and not self-explanatory, this probably needs a `{# l10n: ... #}` comment above to help translators.